Modules/Mobius/MITMobiusDetailContainerViewController.m

```diff
@@ -3,40 +3,62 @@
 
 @interface MITMobiusDetailContainerViewController () <UIPageViewControllerDataSource, UIPageViewControllerDelegate>
 
-@property (strong, nonatomic) NSArray *resources;
-
 @property (strong, nonatomic) UIPageViewController *pageViewController;
 @property (strong, nonatomic) NSArray *mainLoopCycleButtons;
 
-@property (strong, nonatomic) MITMobiusResource *currentResource;
+@property (nonatomic) NSUInteger currentIndex;
 
-@property (nonatomic) BOOL isTransitioning;
+@property (nonatomic,getter=isTransitioning) BOOL transitioning;
 @property (nonatomic, strong) NSMutableArray *inputViews;
+@property (nonatomic,getter=isPagingEnabled) BOOL pagingEnabled;
+
+@property (nonatomic,weak) UIBarButtonItem *pagingBarButtonItem;
+@property (nonatomic,weak) UIButton *nextPageButton;
+@property (nonatomic,weak) UIButton *previousPageButton;
 
 @end
 
 @implementation MITMobiusDetailContainerViewController
+- (instancetype)initWithNibName:(NSString *)nibNameOrNil bundle:(NSBundle *)nibBundleOrNil
+{
+    self = [super initWithNibName:nibNameOrNil bundle:nibBundleOrNil];
```

Comment:
Investigating why "self = [super initWithNibName:nibNameOrNil bundle:nibBundleOrNil];"
makes the view appear behind the navigation bar.  When commented out view appear below navigation bar

@bskinner @justinanderson